oeps/architectural-decisions/oep-0052-arch-event-bus-architecture.rst

```diff
@@ -103,6 +102,11 @@ Consequences
 Change History
 **************
 
+2022-11-29
+==========
+
+* Removed disclaimer about provisional ADRs (most have since been accepted, any that are still provisional are marked as such)
```

Comment:
You also added the reference for follow-up work.

Reply:
[fixed]